server/storage/memory.go

```diff
@@ -158,6 +196,25 @@ func (st *MemStorage) Delete(gun string) error {
 	return nil
 }
 
+// GetChanges returns a []Change starting from but excluding the record
+// identified by changeID. In the context of the memory store, changeID
+// is simply an index into st.changes. The ID of a change, and its index
+// are equal, therefore, we want to return results starting at index
+// changeID+1 to match the exclusivity of the interface definition.
+func (st *MemStorage) GetChanges(changeID string, pageSize int, filterName string, reversed bool) ([]Change, error) {
+	id, err := strconv.ParseInt(changeID, 10, 32)
```

Comment:
`ParseUint` here too?  Generally wondering if we should change `pageSize` to a `uint` in signatures too